pyEpiabm/pyEpiabm/intervention/abstract_intervention.py

```diff
@@ -0,0 +1,45 @@
+#
+# AbstractIntervention Class
+#
+
+
+class AbstractIntervention:
+    """Abstract class for Interventions.
+
+    """
+    def __init__(self, start_time, policy_duration, threshold, population):
+        """Set the parameters of the interventions
```

Comment:
Style wise we typically have a full stop after the initial description of the method, but not after any parameter descriptions
```suggestion
        """Set the parameters of the interventions.
```